website/content/api-docs/features/consistency.mdx

```diff
@@ -2,50 +2,277 @@
+other than the default, include the desired consistency mode as a URL query parameter
+when calling the endpoint:
+- `stale`: Use the `stale` query parameter
+- `consistent`: Use the `consistent` query parameter
+- `default`: Use the `leader` query parameter;
+   only relevant [if the default consistency mode is changed to `stale`](#changing-the-default-consistency-mode-advanced-usage)
+
+#### Changing the Default Consistency Mode (Advanced Usage)
+
+~> **Caution**:
+  Configuring [`discovery_max_stale`] may result in a prolonged outage
+  if your Consul servers become overloaded.
+  We do not recommend using this setting to limit staleness
+  unless it would be preferable for your services to be unavailable
+  than to receive results that are too stale.
+  If bounded result consistency is required by a service, consider
+  modifying the service to use `consistent` mode
+  [on a per-request basis](#overriding-a-request-s-consistency-mode).
+
+The default consistency mode can be changed to `stale` for service discovery HTTP API endpoints,
+including:
+- [Catalog API](/api-docs/catalog)
+- [Health API](/api-docs/health)
+- [Prepared Query API](/api-docs/query)
+
+This allows Consul operators to spread service discovery read load across Consul servers
+with a centralized configuration change, avoiding the need to modify every service to
+explicitly set `stale` consistency mode in their requests.
+Services can still override this default on a per-request basis by
+[specifying a supported consistency mode as a query parameter in the request](#overriding-a-request-s-consistency-mode).
+
+To configure Consul servers that receive service discovery requests to use `stale`
+consistency mode unless overriden,
+set [`discovery_max_stale`] to a value greater than zero in their agent configuration.
+The `stale` consistency mode will be used by default unless the data is sufficiently stale:
+its Raft log's index is more than [`discovery_max_stale`] indices behind the leader's.
+Setting a positive value for [`discovery_max_stale`]
+is analogous to [limiting the staleness of Consul DNS queries](#limiting-staleness-advanced-usage)
+by setting [`dns_config.max_stale`].
+
+A common cause for response staleness exceeding [`discovery_max_stale`] is that
+Consul servers are overloaded by requests. Under these circumstances, [`discovery_max_stale`]
+will make the overloading worse by converting cheap `stale` reads to more expensive `default` reads.
+
+### Visibility into Response Staleness
 
 To support bounding the acceptable staleness of data, responses provide the
 `X-Consul-LastContact` header containing the time in milliseconds that a server
 was last contacted by the leader node. The `X-Consul-KnownLeader` header also
 indicates if there is a known leader. These can be used by clients to gauge the
 staleness of a result and take appropriate action.
+
+### Visibility into Consistency Mode Used
```

Comment:
@banks, @trujillo-adam : this response header wasn't previously documented, so I decided to add it here (along with an example).